### PR TITLE
Add "Load build log" button to build page if log is absent.

### PR DESCRIPTION
**Pull request by rmmh**
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/673)

### CODE REVIEW

File: gubernator/templates/build.html [hunk 1]
```diff
@@ -49,7 +49,9 @@
 		% for name, time, text, filename in failures
 			<hr>
 			<h3><a class="anchor" id="{{name|slugify}}" href="#{{name|slugify}}">{{name}}<span class="time"> {{time|duration}}</span></h3></a>
-			<pre class="cmd" onclick="select(this)">{{name | testcmd}}</pre>
+			% if 'junit_runner' not in filename
```
Comment (spxtr) [inline gubernator/templates/build.html]: I think the best way would be if it checked for "e2e.go" classname, but this is probably fine.

File: gubernator/view_base.py [hunk 2]
```diff
@@ -111,13 +111,13 @@ def memcache_memoize(prefix, expires=60 * 60, neg_expires=60):
     namespace = os.environ['CURRENT_VERSION_ID']
     def wrapper(func):
         @functools.wraps(func)
-        def wrapped(arg):
-            key = '%s%s' % (prefix, arg)
+        def wrapped(*args):
```
Comment (spxtr) [inline gubernator/view_base.py]: Why is this change necessary?

Comment (rmmh) [inline gubernator/view_base.py]: It's just a little cleanup. "Fix memcache_memoize decorator to handle multi-argument functions."

File: gubernator/templates/build.html (hunk 1 above)
Comment (rmmh) [inline gubernator/templates/build.html]: It turns out that e2e tests *do* have `classname="Kubernetes e2e suite"`, which is ugly to show in the UI, so I'm just doing it like this instead.

**Comment by rmmh** [comment]
The button being a link is because (conceptually) the button just modifies the state of the current page.

File: gubernator/templates/build.html (hunk 1 above)
Comment (spxtr) [inline gubernator/templates/build.html]: OK, that's fine. I reserve the right to break this at any time.
